NDB.Covid19/NDB.Covid19.Droid/Views/InitializerActivity.cs

```diff
@@ -50,6 +51,9 @@ protected override void OnCreate(Bundle savedInstanceState)
             _launcherButton.Text = InitializerViewModel.LAUNCHER_PAGE_START_BTN;
             
             _launcherButton.Click += new SingleClick(LauncherButton_Click).Run;
+
+            Window.SetStatusBarColor(Android.Graphics.Color.ParseColor("#32345C"));
```

Comment:
Should we use the resource color? for instance like: Window.SetStatusBarColor(new Color(ContextCompat.GetColor(this, Resource.Color.colorPrimary)));

Reply:
Yes, that's better! Updated